src/core/reducers/search.js

```diff
@@ -13,19 +13,18 @@ export const SEARCH_STARTED: 'SEARCH_STARTED' = 'SEARCH_STARTED';
 export const SEARCH_LOADED: 'SEARCH_LOADED' = 'SEARCH_LOADED';
 
 const SEARCH_ABORTED: 'SEARCH_ABORTED' = 'SEARCH_ABORTED';
-const SEARCH_RESET: 'SEARCH_RESET' = 'SEARCH_RESET';
 
 export type SearchState = {|
   count: number,
-  filters: SearchFilters | {},
+  filters: SearchFilters | {} | null,
```

Comment:
Can't it just be this?

```suggestion
  filters: SearchFilters | null,
```

I don't see any Flow errors.

I mean, it doesn't really matter since `SearchFilters` can be an empty object.

Reply:
thanks